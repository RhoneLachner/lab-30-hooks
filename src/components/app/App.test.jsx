import React from 'react';
import { render, cleanup, act } from '@testing-library/react';
import App from './App';
act(() => {
  describe('App component', () => {
    afterEach(() => cleanup());
    it('renders App', () => {
      const { asFragment } = render(<App />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
