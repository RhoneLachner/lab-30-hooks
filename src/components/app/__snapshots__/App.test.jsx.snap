// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <h1>
      loading
    </h1>
  </div>
</DocumentFragment>
`;
